Sources/TuistSupport/System/System.swift

```diff
@@ -585,17 +585,23 @@ public final class System: Systeming {
         try process.launch()
     }
 
+    var cachedSwiftVersion: String?
```

Comment:
You might want to add some locks to prevent race conditions when accessing it from multiple threads

Reply:
Nice catch, done! I noticed we have @Atomic property wrapper in Tuist Support, that should do the work :)

Reply:
To help stress test this you could try generating a fixture with 50 projects :)